src/main/scala/com/dotdata/sbt/SbtConfigPlugin.scala

```diff
@@ -195,10 +195,16 @@ object SbtConfigPlugin extends AutoPlugin {
           Test / console / scalacOptions --= scalacOptionsConsoleExclusions
         )
 
+      val scalastyleSettingsDef: Def.SettingsDefinition =
+        if (scalastyleEnabled) scalastyleSettings(scalastyleExcludes)
+        else Seq(Test, Compile).flatMap{ inConfig(_) {
+          scalastyle := {}
+        }} :+ (scalastyle := {})
```

Comment:
This is long enough / complex enough that I'd recommend using curly braces on the if-else and going multiline.  I also think avoiding the `inConfig(_)` flatMap would be easier to read:
```suggestion
        if (scalastyleEnabled) {
          scalastyleSettings(scalastyleExcludes)
        } else {
          Seq(
            Test / scalastyle := {}, 
            Compile / scalastyle := {},
            scalastyle := {}
          )
        }
```

Reply:
👍